origin-discovery/src/apollo/resolvers.js

```diff
@@ -22,94 +22,99 @@ function relatedUserResolver (walletAddress, info) {
 }
 
 // Resolvers define the technique for fetching the types in the schema.
-const resolvers = {
-  JSON: GraphQLJSON,
-  Query: {
-    async listings (root, args, context, info) {
-      // TODO: handle pagination (including enforcing MaxResultsPerPage), filters, order.
-      // Get listing Ids from Elastic.
-      const { listingIds, stats } = await search.Listing
-        .search(
-          args.searchQuery,
-          args.filters,
-          args.page.numberOfItems,
-          args.page.offset,
-          true // idsOnly
-        )
-      // Get listing objects from DB based on Ids.
-      const listings = await getListings(listingIds)
+const getResolvers = function (listingInfo) {
```

Comment:
Passing listingInfo works but it might be simpler to just access the listingInfo global variable ? (I think it would simplify the code a bit).
That's anyway what the updateHiddenFeaturedListings method already does...